src/components/FileInputField/README.md

```diff
@@ -162,6 +172,40 @@ It's possible to disable the whole input.
 <FileInputField
   disabled
   label="Attachment"
+  onFilesChanged={() => {}}
+/>
+```
+
+## Handling Files
```

Comment:
This is good for me! 

I am just not sure about `can be handled`. It must be/have to be handled as it is required prop.